packages/xchain-bitcoincash/package.json

```diff
@@ -34,22 +34,20 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@types/bitcore-lib-cash": "^8.23.2",
+    "@psf/bitcoincashjs-lib": "^4.0.2",
+    "@types/bchaddrjs": "0.4.0",
     "@xchainjs/xchain-client": "^0.6.0",
     "@xchainjs/xchain-crypto": "^0.2.3",
-    "bitcore-mnemonic": "^8.24.2",
     "axios": "^0.21.0",
     "axios-mock-adapter": "^1.19.0",
-    "bitcore-lib-cash": "^8.24.2"
+    "bchaddrjs": "0.5.2"
```

Comment:
Do we need to stick to "0.5.2" or can we use `"^0.5.2"` (note `^`)

Add `bchaddrjs` to `peerDependencies`, too